tasks/sep/metal-002-fp-upgrade/input.json

```diff
@@ -0,0 +1,607 @@
+{
+  "version": "1.0",
+  "chainId": "11155111",
+  "createdAt": 1746479428586,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.17.0",
+    "createdFromSafeAddress": "",
+    "createdFromOwnerAddress": ""
+  },
+  "transactions": [
+    {
+      "to": "0xF7Bc4b3a78C7Dd8bE9B69B3128EEB0D6776Ce18A",
+      "value": "0",
+      "data": "0x99a88ec400000000000000000000000001d4dfc994878682811b2980653d03e589f093cb000000000000000000000000d81f43edbcacb4c29a9ba38a13ee5d79278270cc",
+      "contractMethod": {
+        "inputs": [
+          {
+            "name": "_proxy",
+            "type": "address",
+            "internalType": "address payable"
+          },
+          {
+            "name": "_implementation",
+            "type": "address",
+            "internalType": "address"
+          }
+        ],
+        "name": "upgrade",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "_proxy": "0x01D4dfC994878682811b2980653D03E589f093cB",
+        "_implementation": "0xd81f43edbcacb4c29a9ba38a13ee5d79278270cc"
```

Comment:
Looks like the portal implementation is at the wrong version?:
```
% cast call 0xd81f43edbcacb4c29a9ba38a13ee5d79278270cc "version() (string)"          
"1.1.0"
% cast call 0xd81f43edbcacb4c29a9ba38a13ee5d79278270cc "respectedGameType() (uint32)"
Error: server returned an error response: error code 3: execution reverted, data: "0x"
```

I think this the portal is supposed to be at `3.10.0` [according to the 1.4.0 release notes](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0)

Reply:
That address is the StorageSetter contract. It allows the next couple of transactions to edit the storage to clear the initialised flags.  Then it is upgraded to the actual portal implementation at `0x35028bae87d71cbc192d545d38f960ba30b4b233` which is the correct implementation for portal in contracts releases v1.6.0 and v1.8.0. So this is fine.